plugins/nodes/src/js/columns/NodesTableMemBarColumn.tsx

```diff
@@ -3,12 +3,18 @@ import Node from "#SRC/js/structs/Node";
 // TODO: DCOS-39079
 // import { IWidthArgs as WidthArgs } from "@dcos/ui-kit/packages/table/components/Column";
 import { IWidthArgs as WidthArgs } from "#PLUGINS/nodes/src/js/types/IWidthArgs";
+import ProgressBar from "#SRC/js/components/ProgressBar";
 
 export function membarRenderer(data: Node): React.ReactNode {
-  // TODO: DCOS-38823
-  return <span>{data.get("used_resources").mem.toString()}</span>;
+  return (
+    <ProgressBar
+      data={[{ value: data.get("used_resources").mem, className: "color-1" }]}
+      total={100}
+    />
+  );
 }
+
 export function membarSizer(args: WidthArgs): number {
-  // TODO: DCOS-38823
-  return Math.max(100, args.width / args.totalColumns);
+  // TODO: DCOS-39147
+  return Math.min(60, Math.max(60, args.width / args.totalColumns));
```

Comment:
Isn't this the same as saying x is 60 <= x <= 60? Wouldn't the `Math.max` be enough if you want to express that it has at least the width 60?

Reply:
I was thinking that I could just return 60, but it was done with Math.min and Math.max in the CPU column, so there must be some reason for that.

Reply:
https://jira.mesosphere.com/browse/DCOS-39147 - the jira related to this

Reply:
Oh no, that must have slipped through while reviewing 🙈 @juliangieseke What is your stance on this, should we just return 60 or what?

Reply:
hm, yep 🤷‍♂️  but lets not change this here please, there is a follow-up which touches these anyway (see linked todo ticket)

Reply:
updated description of DCOS-39147